frontend/src/common/queries/collections.ts

```diff
@@ -40,7 +40,7 @@ export function useCollection(id: string) {
   return useQuery<Collection>([USE_COLLECTION, id], fetchCollection);
```

Comment:
ignore this file! I'm hard coding `createCollection()` to return a collection id for demo lol Once BE API is ready, I'll update this!

Reply:
😂